test/blackbox-tests/test-cases/pkg/gh10985.t

```diff
@@ -50,10 +50,7 @@ Now we set up a lock file with this package and then attempt to use it:
   > EOF
 
   $ dune build @ocaml-index
-  Error: This rule defines a directory target "default/.pkg/mypkg/target" that
-  matches the requested path
-  "default/.pkg/mypkg/target/lib/mypkg/lib/cctx.ocaml-index" but the rule's
-  action didn't produce it
-  -> required by _build/default/.foo.objs/cctx.ocaml-index
-  -> required by alias ocaml-index
+  File ".foo.objs/_unknown_", line 1, characters 0-0:
+  Error: Program ocaml-index not found in the tree or in PATH
+   (context: default)
```

Comment:
This test needs `(deps %{bin:ocaml_index})`.
One option could be to move it to the sub-directory `ocaml-index` that specify this dependency for this sub-tree.

Reply:
I added a mock binary here. This tests needs to be part of package management as well unfortunately